packages/ui/__stories__/TextField.stories.module.scss

```diff
@@ -1,3 +1,10 @@
+@use '../styles/constants' as c;
+
 .field {
   margin-bottom: 16px;
 }
+
+.shortField {
+  width: c.$grid * 25;
```

Comment:
I think it should be done on parent element in app not in hive. What if I have to use it with bigger numbers (billions e.t.c)?

Reply:
It's not in Hive, it's only for presentation (at Story)